framework/Web/UI/TTemplate.php

```diff
@@ -441,6 +442,32 @@ protected function configureProperty($component, $name, $value)
 				}
 			}
 			$setter = 'set' . $name;
+			if (method_exists($component, $setter)) {
+				if ($reflector = new \ReflectionClass($component)) {
```

Comment:
The setter/property may not be in the component but in a behavior modifying the component. looping through each behavior would be needed too but that requires #826.

Reply:
Looping all the behaviors from here sounds time expensive (but can be fixed with some caching) but also a bit incorrect from an OOP point of view. 
Maybe adding a way to get the reflected method from inside TComponent itself could be an idea. This would also move the reflection cache inside the class itself, so that other callers can benefit from it.